devops/repos/test_installation.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash 
+set -e 
+DIR=`pwd`
+rm -rf /tmp/hipSYCL-test
+git clone https://github.com/illuhad/hipSYCL.git /tmp/hipSYCL-test
+mkdir -p /tmp/hipSYCL-test/tests/build hipSYCL-test
+rm -rf /tmp/build
+mkdir -p /tmp/build
+
+#ubuntu 18.04 for has insufficient cmake version for building the tests
+#TODO:Install recent cmake in ubuntu 18.04
+for distro in archlinux-rolling centos-7 #ubuntu-18.04 
+do
+echo "Build and comp test for omp backend for $distro"
+SINGULARITYENV_HIPSYCL_PLATFORM=omp singularity exec $HIPSYCL_PKG_CONTAINER_DIR/hipsycl-$distro   cmake \
+    -DCMAKE_PREFIX_PATH=/opt/hipSYCL/boost/boost -S /tmp/hipSYCL-test/tests \
+    -B /tmp/build/
+SINGULARITYENV_HIPSYCL_PLATFORM=omp singularity exec -H /tmp/build $HIPSYCL_PKG_CONTAINER_DIR/hipsycl-$distro \
```

Comment:
Maybe use the new `HIPSYCL_TARGETS` mechanism instead of old `HIPSYCL_PLATFORM`/`HIPSYCL_GPU_ARCH`?

Reply:
I'll have a look